tests/integration/targets/dcnm_policy/tests/dcnm/dcnm_policy_delete.yaml

```diff
@@ -106,28 +106,7 @@
         that:
           - 'item["RETURN_CODE"] == 200'
           - 'item["MESSAGE"] == "OK"'
-      when: (my_idx < (result["diff"][0]["deleted"] | length))
```

Comment:
Can you add test cases to explicitly cover the following:

* Create several policies with a template1 (no variables)
* Add new policies using template1 where `description` and `priority` are different then existing policies
* Force creation of new policies with the force flag set to true
* Verify policies are not created when the force flag is false and policies match already existing policies
* Repeat this for template2 (with variables)

Reply:
Some of these might already be covered but want to make sure we have coverage for all of them

Reply:
Mike these are already covered as part of some of the other cases. Anyway I added some more cases to explicitly cover the new cases. Pushed the changes again.